runtime-manager/modules/ROOT/pages/deployment-strategies.adoc

```diff
@@ -53,393 +58,167 @@ Icons appear at the top of each page in the Runtime Manager documentation, indic
-+
-CloudHub provides a preconfigured default object store that you can reference using Anypoint Connector for Object Store.
-+
-See xref:managing-application-data-with-object-stores.adoc[Manage Application Data Storage with Object Stores].
-+
-One limitation of the CloudHub object store is that idempotent routers work only with in-memory stores.
+=== Scheduling
 
-* *Scheduling*
-+
-You can manage schedules for CloudHub apps using the Runtime Manager UI.
-+
-See xref:managing-schedules.adoc[Manage Schedules].
+[%header,cols="15,40,40,20"]
+|===
+|Environment | Implementation       | Note | Reference
+|CloudHub 2.0 | You can use Runtime Manager to view and control the Scheduler components within the flows in your deployed applications without changing your running application. | To view the application schedules, you must have the Exchange Viewer permission, in addition to the Read Applications permission.| xref:cloudhub-2::ch2-manage-schedules.adoc[Managing App Schedules].
+|CloudHub 1.0 |You can use Runtime Manager to view and control the Scheduler components within the flows in your deployed applications.| You cannot manage CloudHub schedules using scripts. Use the *Schedules* tab or the CloudHub API to manage CloudHub schedules.|xref:managing-schedules.adoc[Manage Schedules].
+|Hybrid |Not available for hybrid deployments. | To schedule tasks, use the Scheduler endpoint element in your flows. | xref:mule-runtime::scheduler-concept.adoc[Scheduler Endpoint].
+|Private Cloud Edition | Not available for Anypoint Platform PCE deployments | To schedule tasks, use the Scheduler endpoint element in your flows. | xref:mule-runtime::scheduler-concept.adoc[Scheduler Endpoint].  
+|Runtime Fabric | Not available for Runtime Fabric deployments. | To schedule tasks, use the Scheduler endpoint element in your flows. | xref:mule-runtime::scheduler-concept.adoc[Scheduler Endpoint].  
+|===
 
-* *Security updates*
-+
-When a security update is available for the runtime version for an app, you can see and apply the update in Runtime Manager.
+=== Security Updates
 
-* *Shared resource support*
-+
-Because each application deployed to CloudHub runs on a separate virtual server, you don't need to use domains to share ports or other resources between apps.
+[%header,cols="15,60"]
+|===
+|Environment | Implementation 
+|CloudHub 2.0 | CloudHub 2.0 regularly applies security patches as needed to ensure that your application is secure. See xref:cloudhub-2::ch2-patch-updates.adoc[CloudHub 2.0 Operating System Patch Updates].
+|CloudHub 1.0 | CloudHub applies security patches as needed to ensure that your application is secure and, once per month, it updates Mule to maintain the stability of your application. See xref:cloudhub-app-runtime-version-updates.adoc[CloudHub Runtime Continuous Updates].
+|Hybrid| After an application is deployed and running, you must apply any security updates for the selected runtime version manually. See <<Security for Hybrid Deployments>>.
```

Comment:
Here's an example of an intra-topic link. I think our best practice for this is to define the link with [[]] and then use <<>> to reference the link you defined.

https://confluence.internal.salesforce.com/display/MTDT/File+and+Anchor+Naming+Conventions#FileandAnchorNamingConventions-AnchorNames